ts/screens/messages/MessageDetailScreen.tsx

```diff
@@ -152,9 +151,12 @@ export class MessageDetailScreen extends React.PureComponent<Props, never> {
   };
 
   private setMessageReadState = () => {
-    const { potMessage, messageUIStates } = this.props;
+    const { potMessage, maybeUIStates } = this.props;
 
-    if (pot.isSome(potMessage) && !messageUIStates.read) {
+    if (
+      pot.isSome(potMessage) &&
+      maybeUIStates.fold(false, uiStates => !uiStates.read)
```

Comment:
no very clear the intended behavior here, maybe `getOrElse` would be easier to understand?

Reply:
It is the equivalent of
```
    if (
      pot.isSome(potMessage) &&
      maybeUIStates.isSome() &&
      !maybeUIStates.value.read
    ) 

is this form clearer?
```

Reply:
Changed as above